apps/dashboard/src/components/app/index.js

```diff
@@ -22,6 +23,10 @@ const allowedRoutes = /^\/(project)\/.*/i;
 const NotFound = () => <div className="app__not-found">404</div>;
 
 const App = () => {
+	useStylesheet( '/ui/stable/theme-compatibility/crowdsignal-editor.css' );
```

Comment:
Not sure about adding this one here. 
Tried this aiming to fix c/uprHKmOb-tr.
The Leven theme has some `font-family` definitions and, loading the theme only for the Editor is causing the fonts to be different when switching to the results tab.

Reply:
Right - you're right about the issue, unfortunately that's just how the themes work, also in WordPress land.  
Like you mentioned, since it's only affecting the editor view and our UI is quite limited there outside of the editor itself, I think in this case it'd be ok to just be extra-explicit with styles on `editor/masterbar` and `editor/project-navigation` so they override whatever the theme comes with.

For now let's make sure we've got leven covered, and I guess we might have to update it once in a while once we start introducing new themes.